crates/autopilot/src/solvable_orders.rs

```diff
@@ -636,6 +650,11 @@ fn get_orders_with_native_prices(
             }
         }
     });
+    additional_tokens.iter().for_each(|token| {
```

Comment:
I think by now we can stop worrying about unused native prices. I assume this was just to reduce the noise from the auction but it's already so big that nobody will notice a few prices more or less. Plus your latest changes made it so that we only very rarely filter out orders due to missing prices in the first place.
And without that price filtering depending on the orders this line can be dropped completely when we simply return all the prices from the estimator.